### PR TITLE
feat: add point and query result struct

### PR DESCRIPTION
**Pull request by shoothzj**

### CODE REVIEW

File: opengemini/point.go [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+type Point struct {
+	Measurement string
+	Precision   string
+	Time        time.Time
+	Tags        map[string]string
+	Fields      map[string]interface{}
+}
+
+func (p *Point) AddTag(key string, value string) {
+	p.Tags[key] = value
```
Comment (xuthus5) [inline opengemini/point.go:14]: check Tags is empty?

File: opengemini/point.go [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+func (p *Point) AddField(key string, value interface{}) {
+	p.Fields[key] = value
```
Comment (xuthus5) [inline opengemini/point.go:18]: ditto

Comment (shoothzj) [reply opengemini/point.go:24]: @xuthus5 Fixed

File: opengemini/point.go (hunk 1 above)
Comment (shoothzj) [reply opengemini/point.go:17]: @xuthus5 Fixed

**Review by xuthus5 — APPROVED** [review]

**Pull request merged**